dotnet/src/webdriver/HttpRequestData.cs

```diff
@@ -31,12 +31,11 @@ public class HttpRequestData
         /// <summary>
         /// Initializes a new instance of the <see cref="HttpRequestData"/> type.
         /// </summary>
-        /// <param name="method">The method of the HTTP request.</param>
-        /// <param name="url">The URL of the HTTP request.</param>
-        /// <param name="postData">The POST data of the HTTP request.</param>
-        /// <param name="headers">The headers of the HTTP request.</param>
-        /// <param name="requestId">The ID of the HTTP request.</param>
-        public HttpRequestData(string method, string url, string? postData, Dictionary<string, string> headers, string requestId)
+        public HttpRequestData()
+        {
+        }
+
+        internal HttpRequestData(string method, string url, string? postData, Dictionary<string, string> headers, string requestId)
```

Comment:
Keep nullability consistent.

Reply:
All the parameters except postData are not null. I wish we could make this non-nullable. Alas.

```suggestion
        internal HttpRequestData(string? method, string? url, string? postData, Dictionary<string, string>? headers, string? requestId)
```